DataStructure/array/hw1.php

```diff
@@ -0,0 +1,97 @@
+<?php 
+
+
+class CalcValues
+{
+    public $arr;
+    public $avg;
+    public $sum;
+    public $min;
+    public $max;
+    public $size;
+
+    function __construct($arr)
+    {
+        if ( gettype($arr) != "array" || $arr == []) {
+            echo "<br>";
+            echo "Incorrect input";
+            $this->arr = [];
+            return; 
+        }
+        
+        $this->arr = $arr;
+        $this->size = sizeof($arr);
+        $this->sum = 0; 
```

Comment:
It's very good to exercise OOP here :)
Usually, we don't put all logic into constructor. Only init some value, please move calculation to a function